src/modules/stake-polygon/actions/fetchAPY.ts

```diff
@@ -0,0 +1,20 @@
+import { RequestAction } from '@redux-requests/core';
+import BigNumber from 'bignumber.js';
+import { createAction as createSmartAction } from 'redux-smart-actions';
+import { PolygonSDK } from '../api/PolygonSDK';
+
+export const fetchAPY = createSmartAction<RequestAction<BigNumber, BigNumber>>(
+  'polygon/fetchAPY',
+  () => ({
+    request: {
+      promise: (async (): Promise<BigNumber> => {
+        const sdk: PolygonSDK = PolygonSDK.getInstance();
```

Comment:
Why just not like that?
```suggestion
        const sdk = PolygonSDK.getInstance();
```